lib/ansible/modules/crypto/openssl_privatekey.py

```diff
@@ -103,7 +103,7 @@
         version_added: "2.4"
     cipher:
         description:
-            - The cipher to encrypt the private key. (cipher can be found by running `openssl list-cipher-algorithms`)
+            - The cipher to encrypt the private key. (cipher can be found by running `openssl list -cipher-algorithms`)
```

Comment:
I am getting error on Mac - 
```
openssl list -cipher-algorithms
openssl:Error: 'list' is an invalid command.
```
whereas, `openssl list-cipher-algorithms` works.

Reply:
That's very interesting. @Akasurde which version of OpenSSL do you have? I've tested it both with 1.1.0j and 1.1.1b; for both only `openssl list -cipher-algorithms` works.

Reply:
From https://sandilands.info/sgordon/simple-introduction-to-using-openssl-on-command-line, it looks like behavior was different for 1.0.1; 0.9.8 apparently doesn't support any variant of this at all.

Reply:
OSX Mojave's openssl (which is LibreSSL 2.6.5) also doesn't like the new syntax.

Reply:
How about:
```suggestion
            - The cipher to encrypt the private key. (Valid values can be found by running `openssl list -cipher-algorithms` or `openssl list-cipher-algorithms`, depending on your OpenSSL version.)
```